FODO_Test_Lattice/Phase_Error_Propagation/README.md

```diff
@@ -0,0 +1,29 @@
+# Phase Error Propagation
+
+## Motivation 
+
+This small study aims to investigate how phase errors, 
+determined by the errors on alpha and beta of the boundary conditions,
+propagate through the lattice.
+
+In particular, this is a test whether the first plus-sign in 
+Eq. (2) of 10.18429/JACOW-IPAC2015-MOPJE054  
+( http://jacow.org/ipac2015/doi/JACoW-IPAC2015-MOPJE054.html )
+should in fact be a minus sign:
+Suitable alpha0 and beta0, means and standard deviations, 
+are choosen and realized as boundary conditions for the FODO-lattice from 
+gaussian distributions.
+From the resulting phase advance distributions, 
+shifted to zero by the phase advance of the realization with only mean-values, 
+the standard deviation is calculated and compared to
+the analytically propagated results.
+
+The resulting plots show that, indeed, the error propagation formular with
+a negative sign is the correct one.
+![Result for 1000 realizations](result_1000realizations.png "Result for 1000 realizations")
```

Comment:
bit hard to infer what and why you come to the conclusion ..
and additional normal distribution as overflay with sigmas marked or so could help

Reply:
added comment about the STD is represented by the green dashed line